Formula/libspatialite.rb

```diff
@@ -13,6 +13,11 @@ class Libspatialite < Formula
       url "https://raw.githubusercontent.com/Homebrew/formula-patches/27a0e51936e01829d0a6f3c75a7fbcaf92bb133f/libspatialite/sqlite310.patch"
       sha256 "459434f5e6658d6f63d403a7795aa5b198b87fc9f55944c714180e7de662fce2"
     end
+
+    patch do
```

Comment:
We need this patch submitted to upstream and accepted. We don't actually maintain software here, we just ship it :)

Reply:
Understandable but libspatialite has not been updated since 2015, so I don't have a lot of hope that anything will come up pushing this upstream.

Reply:
This PR cannot be merged if the patch is unreported.

Reply:
Patch will need to be removed, now that liblwgeom dep is gone

Reply:
Sorry about that patch is removed